libs/sdk-backend-spi/src/workspace/execution/results.ts

```diff
@@ -163,6 +164,30 @@ export interface IResultTotalHeader {
  */
 export type IResultHeader = IResultAttributeHeader | IResultMeasureHeader | IResultTotalHeader;
 
+/**
+ * Represents an execution result warning.
+ * (e.g. when execution was executed successfully, but backend didn't take something into the account)
+ *
+ * @public
+ */
+export interface IResultWarning {
+    /**
+     * Unique identifier of the execution warning
+     */
+    warningCode: string;
+
+    /**
+     * Human readable representation of the execution warning, with C-like printf parameter placehodlers
+     * Example: "metric filter on dimension [%s] not applied"
```

Comment:
I'm still missing some relation to the `parameters`. What about adding soemthing like:

> The values for the placeholders are in the `parameters` array in the order they should be used in the placeholders.

?